src/processes/follow-topic-process.js

```diff
@@ -37,36 +38,68 @@ const anon_to_sign_user = async (client, user, targetUser, sourceId = '') => {
   const channel_id = generate_channel_id_for_anon_chat(user, targetUser, 'autoMessage', sourceId);
   const emoji = BetterSocialConstantListUtils.getRandomEmoji();
   const color = BetterSocialConstantListUtils.getRandomColor();
-  const anon_init_data = {
+  const anonInitData = {
     anon_user_info_color_code: color.code,
     anon_user_info_color_name: color.color,
     anon_user_info_emoji_code: emoji.emoji,
     anon_user_info_emoji_name: emoji.name
   };
-  const new_channel = await ChatAnonUserInfo.create({
+  const newChannel = await ChatAnonUserInfo.create({
     channel_id,
     my_anon_user_id: user,
     target_user_id: targetUser,
-    anon_user_info_color_code: anon_init_data.anon_user_info_color_code,
-    anon_user_info_color_name: anon_init_data.anon_user_info_color_name,
-    anon_user_info_emoji_code: anon_init_data.anon_user_info_emoji_code,
-    anon_user_info_emoji_name: anon_init_data.anon_user_info_emoji_name,
+    anon_user_info_color_code: anonInitData.anon_user_info_color_code,
+    anon_user_info_color_name: anonInitData.anon_user_info_color_name,
+    anon_user_info_emoji_code: anonInitData.anon_user_info_emoji_code,
+    anon_user_info_emoji_name: anonInitData.anon_user_info_emoji_name,
     context: 'auto message',
     initiator: user,
     source_id: sourceId
   });
-  return new_channel;
+
+  return newChannel;
 };
+
+async function sendPushNotification({
+  serverChannel,
+  serverMessage,
+  targetUserId,
+  senderUsername,
+  text
+}) {
+  const notificationPayload = {
+    title: senderUsername,
+    body: `${text?.substring(0, 100)}`
+  };
+
+  const dataPayload = {
+    type: 'message.new',
+    channel_id: serverChannel?.id,
+    message: text,
+    message_schema: 'text',
+    attachment: '',
+    created_at: serverMessage?.message?.created_at || new Date().toISOString(),
+    is_annoymous: 'false',
+    priority: 'high',
+    content_available: 'true'
+  };
+
+  sendChatNotificationByChannelMembers([targetUserId], {
+    notification: notificationPayload,
+    data: dataPayload
+  });
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `sendPushNotification` does not handle any potential errors that might occur when calling `sendChatNotificationByChannelMembers`. It's recommended to add error handling to ensure the application can gracefully handle any issues that arise.
```diff
+  try {
      sendChatNotificationByChannelMembers([targetUserId], {
        notification: notificationPayload,
        data: dataPayload
      });
+  } catch (error) {
+    console.error('Error sending chat notification:', error);
+    throw error;
+  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->